docs/source/runningjobs/runningsjob.md

```diff
@@ -0,0 +1,148 @@
+(running-job-with-sjob)=
+# Running Jobs with sjob
+
+The `sjob` command is designed to simplify the process of submitting jobs on the Discovery cluster using SLURM's `srun` and `sbatch` modes. It provides an intuitive interface for requesting resources and managing job submissions.
+
+## Features
+
+This command provides three modes of operation:
+
+| Mode              | Description                                               |
+|-------------------|-----------------------------------------------------------|
+| **Default**  | To quickly start a job with the default settings.         |
+| **Interactive** | To submit the job interactively with customized resource requests. |
+| **Batch**    | To generate and save `sbatch` scripts with specified parameters. |
+
+## Getting Started with sjob Command
+
+To launch an interactive session, use the following command:
+
+:::{code} bash
+sjob
```

Comment:
We will just need to add the "module load modulename" command here. Lets get it in a module this week.

Reply:
Added command to load the module.